esp/scm/ws_dfu.ecm

```diff
@@ -174,7 +174,6 @@ ESPrequest [nil_remove] DFUQueryRequest
     [min_ver("1.26")] string NodeGroup;
     [min_ver("1.30")] string ContentType;
     string LogicalName;
-    string Description;
```

Comment:
I suppose this is OK.  Normally I would say just deprecate it with a "depr_ver", but in this case it may be better that even someone trying to use an old version of the wsdl get an error since it will no longer behave the same.